### ISSUE TITLE
Drought deciduous trees and changing soil moisture initialization

### ISSUE THREAD

**Issue by rosiealice** [NGEET/fates#668]
So, in my ongoing quest to make the DDTs perform non-ridiculously (hurray or scientific goal)  I realized that the joint phenology/hydrology system is being fundamentally thrown off by the very dry initialization of soil moisture, which is set as VWC of 0.15 in the init_cold routine here:

https://github.com/ESCOMP/CTSM/blob/e33b4658f6c51f8d1cce4ed1986bc3dc28014c5d/src/biogeophys/WaterStateType.F90#L773

What happens after the initialization is that 
1. The soil is dry, and so the leaves come off and stay off for a long time
2. when they come back on, they've lost so much carbon that they die off (of carbon starvation and then termination). 
3. when the few survivors try to regrow, they can't pull the soil moisture down far enough to trigger the normal drought deciduous behaviour and get stuck in a perma-wet situation
4.  in the perma-wet situation, they are further penalized for there being no dry season. 
5.  they also have extremely low GPP, LAI, biomass etc. 

In this way, most of the savanna areas have been dying in large semi-arid areas of my simulations.  e.g. 
https://github.com/rosiealice/global_pft_scripts/blob/master/analysis_scripts/droughtdec_init0.15.ipynb

Changing the initial soil moisture value to 0.85 massively improves things, as shown in:
here:https://github.com/rosiealice/global_pft_scripts/blob/master/analysis_scripts/droughtdec_initw.ipynb

 If you look at the **last** figure in these notebooks, of SMP and VWC across soil layers, you can see that in the 0.15 simulation, the system is still not at equilibrium well into the second year. For many hydrology applications that's fine of course, but for these DDT's with their carbon stores and their mortality thresholds, it really isn't.  

 If you look at the **first** plot (of LAI in each season) to illustrate the impact that changing soil moisture initialization has on the DDT dynamics!

So, my question is, slightly directed at @dlawrenncar  and Sean Swenson (who isn't here, but i'll direct him to this later), is there a reason to **not** modify the initial soil moisture? I can imagine several (flooding, GRACE comparison, permafrost concerns, hydraulic redistribution etc.)  Of course this change can go in with a 'use_fates' caveat, and I see it's already been done for cases with HYDRO, but is there any reason not to do this?  I imagine that this might also be messing a little bit (but not as much, in the absence of mortality and with the magic of XSMR).

Anyway, with this change, everything makes a lot more sense, thankfully...

**Comment by rgknox** [comment]
@rosiealice is part of the problem that FATES is too sensitive to the initial condition? If there is external seed rain, do the new recruits succeed when the drought timers start to trigger again?

**Comment by dlawrenncar** [comment]
@rosiealice I don't think that we have ever really explored the impact of soil water initialization.  It does impact permafrost long term dynamics because the initial soil water state deep in the soil essentially gets frozen in until it thaws at some point in the distant future.  In general, a higher value of initial soil moisture is possibly more realistic.  I'm meeting with Sean tomorrow and I can ask him if he knows of any issues that we should be concerned about. 

Do you need to go so such a high level.  0.85 is way above typical saturation levels since watsat tends to be around 0.3-0.4 for non-organic soils. I think this would mean that there would be a big water flux to the rivers on the first timestep as the soils kick out all the excess water.  You might be able to get same results with something a bit smaller.

**Comment by rosiealice** [comment]
OK cool. I'll try it with a smaller number. I asked Sean and he didn't think it would be problematic... 

Also good point @rgknox. I think that would happen, but also givwn the counters get into some alternative states with soil which is this dry (in deep areas, it needs a lot of water to get back to equilibrium state. So I think I do need to change it...

**Comment by walkeranthonyp** [comment]
Would it be possible to initialise to field capacity, or a fairly high
fraction of?

**Comment by glemieux** [comment]
Pinging @mpaiao on whether your updates in #801  will address this.

**Comment by mpaiao** [comment]
@glemieux I just applied the same assumption done for FATES-Hydro for any FATES simulation, actually following @rosiealice suggestion from this thread. 

I think there was a small confusion in the thread, the value that Rosie suggested (0.85) referred to the value relative to saturation, not the absolute initial soil moisture (m3/m3), so the initial soil moisture is reasonable even for non-organic soils.
